README.md

```diff
@@ -1 +1,61 @@
 # OR-Assistant
+
+## Setup
+
+### Option 1 - Docker
+
+Ensure you have `docker` and `docker-compose` installed in your system.
+
+- **Step 1**: Clone the repository:
+```bash
+  git clone https://github.com/The-OpenROAD-Project/ORAssistant.git
+``` 
+- **Step 2**: Copy the `.env.example` file, and update your `.env` file with the appropriate API keys. Get the [Google Gemini API Key](https://ai.google.dev) and add it to your env file, add other env vars as required.
+```bash
+  cd backend
+  cp .env.example .env
+```
+
+- **Step 3**: Start the server by running the following command,
+```bash
+  docker compose up
+```
+
+### Option 2 - Local Install
+
+- Follow **Step 1** and **Step 2** as mentioned above.
+- **Step 3**: To scrape OR/ORFS docs and populate the `data` folder, run
+```bash
+  python src/tools/scrape_userguide.py
+```
+- **Step 4**: To run the server,
+```bash
+  cd backend
+  python main.py
+```
+
+## Architecture Overview
+
+Currently, documentation from OpenROAD and OpenROAD-flow-scripts is chunked recursively and embedded into FAISS Vector Databases.  
```

Comment:
Are these going to be separated out? OR can be used separately from ORFS.

Reply:
They are not separated currently, but we are exploring options where the architecture can selectively query different data sources depending on the input query.